examples/graph/wgan-graphs.py

```diff
@@ -0,0 +1,545 @@
+"""
+Title: WGAN-GP with R-GCN for the generation of small molecular graphs
+Author: [akensert](https://github.com/akensert)
+Date created: 2021/06/27
+Last modified: 2021/06/27
+Description: Complete implementation of WGAN-GP with R-GCN to generate novel molecules.
+"""
+
+"""
+## References
```

Comment:
Start with an introduction describing what the tutorial teaches you and why it's important. References can go at the end of the introduction.